src/IIIFPresentation/API/Features/Manifest/DlcsManifestCoordinator.cs

```diff
@@ -102,7 +103,7 @@ private static List<JObject> GetAssetJObjectList(WriteManifestRequest request) =
             logger.LogError(exception, "Error creating batch request for customer {CustomerId}, manifest {ManifestId}",
                 customerId, manifestId);
             return EntityResult.Failure(exception.Message, ModifyCollectionType.DlcsException,
-                WriteResult.Error);
+                exception.StatusCode == HttpStatusCode.BadRequest ? WriteResult.BadRequest : WriteResult.Error);
```

Comment:
Maybe not required here but wonder if we need an `HttpStatusCode` -> `WriteResult` converter? We have an approximation of the opposite way in `ControllerBaseX.ModifyResultToHttpResult` but it converts to a full `IActionResult`.

e.g. can batch creation return a 409? If so it'd be good to return that rather than a 500